src/tests/components/Subscriptions.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { Subscriptions } from '../../components/Subscriptions'
+
+describe('Subscriptions', () => {
```

Comment:
Not sure how useful this test is, I would use a Cypress test for this

Reply:
to quote myself above "there are some tests see `components/PayPalSuccess.test.js` where testing the function within `useEffect` proved difficult... also for the Subscriptions.test.js the second test fails if the first, not-so-good test, isn't present", although I believe in a subsequent `PR` I was able to get the second test to pass without the first